kube_deploy/Uat/deploy.yml

```diff
@@ -97,6 +97,11 @@ spec:
                 secretKeyRef:
                   name: shared-secrets
                   key: webhook_supportteam_reporter_slack
+            - name: S3_BUCKET_NAME
```

Comment:
It doesn't appear that this has been added to the secrets in this environment. We'll need to do this before accepting this PR.

Reply:
@BarryPiccinni has now added this to the environment